### ISSUE TITLE
Elevated circulating creatine concentration

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7357]
**Preferred term label:**
Elevated circulating creatine concentration

**Synonyms**

**Definition (free text, please give PubMed ID)**
An elevation above the normal range of creatine in the blood circulation. 

**Parent term (use hpo.jax.org/app)**
Abnormal circulating creatine concentration HP:0012113

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM: 300352 
PMID:11326334  1/1

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
